### PR TITLE
Remove author from LearningPath serializer

### LINKED ISSUE #328: "author" field in LearningPath serializer: is it necessary?
# Description/Context
The `LearningPath` serializer currently includes an `author` field whose value is equal to the id of the user who created it.  

Is this field going to be used for anything? If so, would `username` or `name` be better?  If not, it should be excluded from the serializer data output.

```
        "learning_path": {
            "id": 1,
            "item_count": 0,
            "author": 1
        },
```

### PR DESCRIPTION
**Pull request by mbertrand**
### What are the relevant tickets?
Closes #328

### Description (What does it do?)
Removes "author" field from LearningPathSerializer

### How can this be tested?
Unit tests should pass.
Create a learning path if you don't already have any (log in as an admin and go to http://localhost:8063/learningpaths/)
Check the serializer output at http://localhost:8063/api/v1/learningpaths.  You should see something like this, with no `author` field present:

```
            "learning_path": {
                "id": 1,
                "item_count": 2
            },
```

### CODE REVIEW

File: learning_resources/serializers.py
Comment (ChristopherChudzicki) [inline learning_resources/serializers.py:1]: I think we can `author` from
```python
            models.LearningPath.objects.create(
                learning_resource=path_resource, author=request.user
            )
```
in `LearningPathResourceSerializer.create`, too, yes?

Comment (mbertrand) [reply learning_resources/serializers.py:1]: Only if we want to stop tracking which user created the LearningPath altogether, in which case the field should be removed from the model as well, not just the serializer.

**Review by ChristopherChudzicki — APPROVED** [review]
👍

**Review by ChristopherChudzicki — APPROVED** [review]
👍

**Pull request merged**